.gitignore

```diff
@@ -1,3 +1,7 @@
+telex.egg-info*
+*/telex.egg-info/*
+build/*
+dist/*
 *.pyc
-.venv/
```

Comment:
The team has largely switched to using virtualenvwrapper and we typically keep the envs outside of the project (we use to all keep them in `.venv`). But, still valid point, might as well leave it.